sample.env

```diff
@@ -40,3 +40,7 @@ START_PORT=10000
 END_PORT=10500
 
 MESSAGE_TO_SEND="Lorem ipsum dolor sit amet consectetur, adipiscing elit nisi aptent rutrum varius, class non nullam etiam. Ac purus donec morbi litora vivamus nec semper suscipit vehicula, aliquet parturient leo mollis in mauris quis nisi tincidunt, sociis accumsan senectus pellentesque erat cras sociosqu phasellus augue, posuere ligula scelerisque tempus dapibus enim torquent facilisi. Imperdiet gravida justo conubia congue senectus porta vivamus netus rhoncus nec, mauris tristique semper feugiat natoque nunc nibh montes dapibus proin, egestas luctus sollicitudin maecenas malesuada pharetra eleifend nam ultrices. Iaculis fringilla penatibus dictumst varius enim elementum justo senectus, pretium mauris cum vel tempor gravida lacinia auctor a, cursus sed euismod scelerisque vivamus netus aenean. Montes iaculis dui platea blandit mattis nec urna, diam ridiculus augue tellus vivamus justo nulla, auctor hendrerit aenean arcu venenatis tristique feugiat, odio pellentesque purus nascetur netus fringilla. S."
+
+# If you want to use a different pickup strategy, you can set it here
+# The default is 'implicit', but you can also use 'pickupv2-live'
+PICKUP_STRATEGY='implicit' # or 'pickupv2-live'
```

Comment:
What about pickupv1 and pickupv2 -- are they no longer supported?  V1 is what is used in the current ACA-Py Mediator, and AFAIK, there is a pickupv2 separate from live.

Reply:
There's pickupv2 which only polls for for messages. Pickupv2Live actually sends the live delivery request to set the mediator to send websockets instead of just responding to the polling.

I don't know much about pickupv1.

I can add an option to use the pickupv2 to use polling. I believe this would only be used if the mediator was set to Queue Only.